envi/cli.py

```diff
@@ -476,6 +478,17 @@ def do_script(self, line):
         locals = self.getExpressionLocals()
         locals['argv'] = argv
 
+        if line.startswith("?"):
```

Comment:
didn't have time to learn them.  once @invisig0th accepts this, then please implement them!